lib/hydra/works/services/generic_file/full_text_extraction_service.rb

```diff
@@ -0,0 +1,33 @@
+module Hydra::Works
+  # Extract the full text from the content using Solr's extract handler
+  class FullTextExtractionService
+    def self.run(generic_file)
+      new(generic_file).extract
+    end
+
+    delegate :original_file, to: :@generic_file
+
+    def initialize(generic_file)
```

Comment:
Can you make the methods below this line private to clarify what's in and not in the public API of this service?